.vscode/launch.json

```diff
@@ -8,11 +8,11 @@
       "type": "node",
       "request": "launch",
       "name": "Debug current open test",
-      "program": "${workspaceRoot}/node_modules/.bin/jest",
-      "cwd": "${workspaceFolder}",
+      "program": "${workspaceRoot}/scripts/jest/debugTests.js",
```

Comment:
Fixing this to make it work in the monorepo--the wrapper file is needed to find the correct jest config for the package